### ISSUE TITLE
Viewing results?

### ISSUE THREAD

**Issue by antithing** [vye16/shape-of-motion#4]
Hi! What format are the 4d results in? 
What were the videos rendered with?

Thanks for making this code available!

**Comment by hangg7** [comment]
Hi, thanks for your interest!

> What format are the 4d results in?

We don't export our model to a specific file format to existing renderer. Instead we just simply save the model checkpoints. I have added a checkpoint [here](https://g.co/kgs/Nm4D9cg) for you to play with. Hope it helps!

> What were the videos rendered with?

We render our videos inside of our interactive nerfview viewer, checkout https://github.com/hangg7/nerfview and https://github.com/vye16/shape-of-motion/blob/main/flow3d/vis/viewer.py

Closing this! Feel free to follow up with any further questions.
